lib/types.nix

```
@@ -113,4 +113,38 @@ rec {
         types.optionDescriptionPhrase (class: class == "noun" || class == "composite") elemType
       }";
     };
+
+  pluginLuaConfig = types.submodule (
+    { config, ... }:
+    {
+      options = {
+        pre = lib.mkOption {
+          type = types.lines;
+          default = "";
+          description = ''
+            Lua code inserted at the start of the plugin's configuration.
+            This is simply sugar around `lib.nixvim.utils.mkBeforeSection content`
```

```suggestion
            This is the same as using `lib.nixvim.utils.mkBeforeSection` when defining `content`.
```

Or maybe something like 

```suggestion
            This is the same as `content = lib.mkOrder ${(lib.nixvim.utils.mkBeforeSection null).priority} ""`
```

?

---

I'd prefer the first one